### DISCUSSION TITLE
Group Permissions

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [requarks/wiki#6111] [Guides]
It could be a JIRA Confluence-like structure.  Wiki.js is very good to manage the authorization system on the group through the path, but it would be nice to **limit the authority** of the people in the group to the page and path. For example, I want only two people to see the documents related to the budget. In this case, instead of creating a new group, it may give a more free structure to be able to organize the authorizations related to that path on an individual basis.

**Comment by ghost** [comment]
Yes this feature would be great. However think about your tone in which you are requesting. Keep in mind that this is an open source project that the lead developer manages on his free time without any monetary compensation. 

Other than that, Wiki.js 3.0 will come with a lot of new features, including UX improvements. I can very well imagine that the distribution of permissions will be improved in that version, too. You can also suggest a feature here: https://js.wiki/support

So lets stay tuned for new updates :)

**Reply by YunusEmreAlps** [reply]
Thank You @Aemilius11. I respect and support this project every time. Yes, I am aware the first paragraph is so aggressive but this coming from this https://github.com/requarks/wiki/discussions/5584 discussion. I will show the parts I took as a reference next time. But I deleted it at this time.

Have a good day!
